packages/smart-vaults/src/library/UserOperationLib.sol

```diff
@@ -113,4 +113,8 @@ library UserOperationLib {
     function hash(PackedUserOperation calldata userOp_) internal pure returns (bytes32) {
         return keccak256(encode(userOp_));
     }
+
+    function unpackUints(bytes32 packed) internal pure returns (uint256 high128, uint256 low128) {
+        return (uint128(bytes16(packed)), uint128(uint256(packed)));
```

Comment:
just for my own understanding. you use `uint128` here, but in the definition it returns two `uint256`. Does that matter? Is there a reason they are different?

Reply:
so I forked this from entry point that handles the packing and unpacking of user operations.
shouldn't matter over here just wanted to keep it consistent with entry point

Reply:
a little wild to me that this code works lol. so `bytes16` takes the top 16 bytes and then the conversion to `uint128` .. pretends like they're not the top 16 bytes?